cmd/get/launch_plan.go

```diff
@@ -55,6 +55,16 @@ Retrieve all the launch plans with filters:
 
   flytectl get launchplan -p flytesnacks -d development --filter.fieldSelector="name=core.basic.lp.go_greet"
 
+Retrieve all active launch plans with filters:
```

Comment:
nit : you can omit the `with filters` in both the lines.

We probably want to add all properties which are filterable in the docs not just for launchplan, but for executions, workflows, task , named entity etc and this should be available easily on docs.flyte.org

Can we create a separate ticket to capture this effort